base_export_async/models/delay_export.py

```diff
@@ -118,37 +111,28 @@ def export(self, params):
             date_today + relativedelta(days=+int(time_to_live))
         )
 
-        # TODO : move to email template
         odoo_bot = self.sudo().env.ref("base.partner_root")
         email_from = odoo_bot.email
         model_description = self.env[model_name]._description
-        self.env["mail.mail"].create(
+
+        export_record.write(
             {
+                "url": url,
+                "expiration_date": expiration_date,
+                "model_description": model_description,
+            }
+        )
+
+        self.env.ref("base_export_async.delay_export_mail_template").send_mail(
+            export_record.id,
+            email_values={
```

Comment:
Was this refactor to a mail template necessary for the migration?

Reply:
For the migration, no. 
Since `# TODO : move to email template` was present, I used what was done in https://github.com/OCA/queue/pull/492/commits/cf7a38ed1f5b50824a3dcd2e5759db55ad0d840e#diff-da16e17b668a488a310b93c6e26ceb527ba53e3b09faf56e9b2e60600aed2506.